interfaces/builtin/desktop_launch.go

```diff
@@ -19,6 +19,16 @@
 
 package builtin
 
+import (
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/snap"
+)
+
+var prioritizedSnippetDesktopFileAccess = apparmor.RegisterSnippetKey("desktop-content-access")
```

Comment:
here we probably want a comment describing a bit why we need this and also two constants to use as priorities instead of the hard coded numbers

Reply:
Good point. Oh, I also forgot to comment that, probably, you will want to rename the variable names 😇

Reply:
Done.